src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -294,6 +296,49 @@ export default function formReducer(state = initialState, action) {
         data: { ...state.data, facilityType: action.facilityType },
       };
     }
+    case FORM_PAGE_FACILITY_V2_OPEN: {
+      return {
+        ...state,
+        childFacilitiesStatus: FETCH_STATUS.loading,
+      };
+    }
+    case FORM_PAGE_FACILITY_V2_OPEN_SUCCEEDED: {
+      let newSchema = action.schema;
+      const facilities = action.facilities.sort((a, b) => {
+        return a.name < b.name ? -1 : 1;
+      });
+
+      newSchema = set(
+        'properties.vaFacility',
+        {
+          type: 'string',
+          enum: facilities.map(facility => facility.id),
```

Comment:
Updated this to pass facilities directly to the component instead of through `formContext`